package.json

```diff
@@ -35,14 +35,15 @@
     "babelify": "^7.2.0",
     "brfs": "^1.4.3",
     "browserify": "^13.0.0",
+    "devtool": "^2.3.1",
     "faucet": "0.0.1",
     "hyperscript": "^1.4.7",
     "nyc": "^8.3.0",
     "query-dom": "^3.0.7",
     "semistandard-deku": "micnews/semistandard#deku",
     "snazzy": "^5.0.0",
+    "tapava": "^2.3.0",
     "tape-catch": "^1.0.4",
```

Comment:
@ryanscottaudio are we using both tapava and tape-catch? Or can tape-catch be removed?